index.js

```diff
@@ -8,7 +8,7 @@ var UglifyJS = require("uglify-js");
 var _paceConfig = {};
 var _defaultPaceConfig = {
   color: 'blue',
-  theme: 'material',
+  theme: 'minimal',
```

Comment:
That's not necessary since recently the material theme was merged into original pace
https://github.com/HubSpot/pace/pull/214